ci/jenkins/test-vmc.sh

```diff
@@ -407,11 +433,19 @@ fi
 if [[ "$TESTCASE" == "integration" ]]; then
     run_integration
 elif [[ "$TESTCASE" == "e2e" ]]; then
+    if [[ "$RUN_TEST_ONLY" == true ]]; then
+        run_e2e
+        exit 0
```

Comment:
Do I miss something? It seems to always return success. In current implementation, it uses `TEST_FAILURE ` to indicate if success or not and exit 0 or 1 with if statement at the end of the script.

Reply:
Yes, the logic here should be modified.

Plus: Do you think that we should continue to do other tests when one of the previous tests fails?

Reply:
I think yes. As long as we can collect enough logs from failed cases and cleanup properly before next one.

Reply:
Logic modified here.